probe-rs-target/src/chip.rs

```diff
@@ -110,10 +117,24 @@ impl Chip {
             default_binary_format: Some(BinaryFormat::Raw),
         }
     }
+
+    /// Returns whether this chip definition is, not counting its name, equal to another chip.
+    pub fn is_duplicate_of(&self, other: &Self) -> bool {
+        self.name.len() == other.name.len()
```

Comment:
This feels like it might be a bit fragile when new chip fields are added, if we miss them here in this check we could inadvertently remove actually different variants

Reply:
How about now? :)